src/views/TradingCompetition/components/BattleCta/index.tsx

```diff
@@ -69,15 +72,16 @@ const BattleCta: React.FC<CompetitionProps> = ({
     <ClaimModal userTradingInformation={userTradingInformation} onClaimSuccess={onClaimSuccess} />,
     false,
   )
-  const { hasRegistered, hasUserClaimed, userCakeRewards, userPointReward, canClaimNFT } = userTradingInformation
-
-  const userCanClaimPrizes = !hasUserClaimed && (userCakeRewards !== '0' || userPointReward !== '0' || canClaimNFT)
```

Comment:
Moved `userCanClaimPrizes`, `finishedAndPrizesClaimed`, `finishedAndNothingToClaim` to TradingCompetition/index level so that the logic can be shared with other components on the page